releasenotes/notes/make_domain_forwarder_chan_buffer_configurable-eb2ed3cb09ff2962.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Make domain forwarder high prio, low prio and requeue channels buffer size configurable
```

Comment:
I'm not sure we want to list this change in the changelog if we're not adding them to the datadog.yaml.example. Customers should not have to tweak those (in 99% case). This might be part of the non documented settings we have.